x-pack/plugins/synthetics/public/legacy_uptime/components/overview/query_bar/translations.ts

```diff
@@ -13,8 +13,8 @@ export const KQL_PLACE_HOLDER = i18n.translate('xpack.synthetics.kueryBar.search
 });
 
 export const SIMPLE_SEARCH_PLACEHOLDER = i18n.translate(
-  'xpack.synthetics.kueryBar.searchPlaceholder.simple',
+  'xpack.synthetics.kueryBar.searchPlaceholder.simpleText',
   {
-    defaultMessage: 'Search by monitor ID, name, or url (E.g. http:// )',
+    defaultMessage: 'Search by monitor ID, name, or url (E.g. http, homepage, google, my monitor)',
```

Comment:
I don't know what type of examples we want to include, and if we want to use a specific company name. @florent-leborgne